GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -332,6 +332,12 @@ public override bool TryGetGVFSEnlistmentRoot(string directory, out string enlis
             return WindowsPlatform.TryGetGVFSEnlistmentRootImplementation(directory, out enlistmentRoot, out errorMessage);
         }
 
+        public override bool KillProcessTree(int processId)

```

Comment:
Should we log that we are killing a process here (or in the caller)? Would the name of the process we are killing be useful if we have to look through the logs?

Reply:
It's always going to 'git' so I wouldn't say the name is useful honestly.  Do log weather the kill was a success or not.

Reply:
This is in the `WindowsPlatform` code, which does not control what process is being killed, and so it could be any `pid`… The only known caller today might only do it for `git.exe`, but this could change in the future. My other thought was that it would be nice to know when we are killing a process - it shouldn't add too much overhead to the logging, as we don't do it very often.

Reply:
I updated it to carry thru processName, exitCode, and error message.  This should give us all the info we need.